### DISCUSSION TITLE
How to fork mainnet to local?

### DISCUSSION THREAD

**Discussion by l2l2l2l** [ConsenSys-archive/truffle#5372] [General Q&A]
Hi, I want to fork mainnet at a specific blockNumber. I don't want to use a RPC provider cuz it may rate limit my request.

Is it possible to "download" the archive@blockNumber locally?

**Comment by davidmurdoch** [comment]
Hi @lilemw Please see the response here: https://github.com/trufflesuite/ganache/discussions/3447
